e2e/interchaintestv8/e2esuite/light_clients.go

```diff
@@ -0,0 +1,344 @@
+package e2esuite
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+
+	ibcwasmtypes "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	mock "github.com/cosmos/ibc-go/v8/modules/light-clients/00-mock"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/testutil"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/ethereum"
+	ethereumligthclient "github.com/srdtrk/solidity-ibc-eureka/e2e/v8/types/ethereumlightclient"
+)
+
+func (s *TestSuite) CreateEthereumLightClient(ctx context.Context, simdRelayerUser ibc.Wallet, ibcContractAddress string) {
+	if s.ethTestnetType == TestnetTypePoW {
+		s.createDummyLightClient(ctx, simdRelayerUser)
+	} else {
+		s.createUnionLightClient(ctx, simdRelayerUser, ibcContractAddress)
+	}
+}
+
+func (s *TestSuite) UpdateEthClient(ctx context.Context, ibcContractAddress string, minimumUpdateTo int64, simdRelayerUser ibc.Wallet) {
+	if s.ethTestnetType == TestnetTypePoW {
+		return
+	}
```

Comment:
switch statement here or:

```suggestion
	if s.ethTestnetType != TestnetTypePoS {
		return
	}
```